### ISSUE TITLE
Meta: Warn users about creating forks of outdated software

### ISSUE THREAD

**Issue by josephbisch** [devrandom/gitian-builder#163]
There seems to have been a lot of recent issues posted by people that are creating forks using some guide that seems to instruct them to create a fork of an old version of Litecoin or something.

They end up with a fork containing Gitian descriptors such as [this descriptor](https://github.com/lemanum/LMN/blob/7a4ce94039ea613dbce578668c801e6d9deb071f/contrib/gitian-descriptors/gitian-win32.yml), which uses Ubuntu Lucid which has been eol since 2013. They end up running into compilation issues and the issues they post in the issue tracker here often lack important information such as which coin they are trying to build. Even when they are successful in building, the use of Ubuntu Lucid in 2018 is questionable, to put it lightly.

So should we put a warning at the top of the README about following outdated guides and using old versions of distros for the builds? Should we use a template for issues so that people are prompted to link to the descriptor they are trying to use if they have an issue concerning a specific project that uses Gitian? Should we do both?

**Comment by devrandom** [comment]
An issue template is likely a good idea.  I'm slightly worried that we are providing tech support outside of the scope of this project, but maybe it's OK if we have the bandwidth and it's not too much of a burden.

**Comment by josephbisch** [comment]
Yeah, I'm also worried about providing tech support outside of Gitian itself. I'm going to stop dealing with issues that are obviously caused by the particular descriptor rather than Gitian itself, because there has been an increase in the number of such issues and the projects are obviously not reading the documentation here and using current examples to model their Gitian usage after.

**Comment by neutcoin-project** [comment]
@josephbisch  I actually got the old descriptor to compile after working with it a bunch. There is a video series on YouTube (that isn't up to date and not documented very well) that shows how to fork Litecoin v0.8.7.4 into a custom cryptocurrency, if you were wondering where the influx of old descriptor issues came from. I would just add a warning against opening issues that stem from old descriptors. I feel kinda bad about opening an issue about an old descriptor, so I'd be willing to provide tech support to any newer issues about these old Litecoin build issues.
